### ISSUE TITLE
Lade-Symbol wenn Like/Dislike gedrückt wird

### ISSUE THREAD

**Issue by xD33m** [tim595/Tinster#41]
evtl. auch ein Übergang auf die nächste SwipeCard ? 😱

**Comment by tim595** [comment]
Hi is this still ongoing? I really need this to be fixed

**Comment by xD33m** [comment]
No. I'm very busy right now, building the next generation of AI agentic blockchain. Please create a PR yourself.
